extensions/amp-autocomplete/amp-autocomplete.md

```diff
@@ -58,16 +58,56 @@ Example:
 
 ```html
 <amp-autocomplete
-  filter="substring"
   id="myAutocomplete"
-  src="/static/samples/json/amp-autocomplete-cities.json"
+  src="{{server_for_email}}/static/samples/json/amp-autocomplete-cities.json"
 >
   <input />
+  <template type="amp-mustache">
+    <div data-value="{{.}}">{{.}}</div>
+  </template>
 </amp-autocomplete>
 ```
 
 [/filter] <!-- formats="email" -->
 
+When using the `src` attribute with `amp-autocomplete`, the response from the endpoint contains data to be rendered in the specified template.
+
+You can specify a template in one of two ways:
+
+- a `template` attribute that references an ID of an existing templating element.
+- a templating element nested directly inside the `amp-autocomplete` element.
+
+For more details on templates, see [AMP HTML Templates](../../spec/amp-html-templates.md).
+
+[tip type="note"]
+Note also that a good practice is to place these in a single top-level element to prevent unintended side effects. Some possible consequences of providing multiple top-level elements: each top-level element could be treated as a distinct item in a list, or all same-level elements could be wrapped by an additional <div> layer, potentially breaking structure-based CSS rules. This means the following input:
```

Comment:
Are you saying that this is an issue for AMP in general and not specific to AMP for Email? I was under the impression that this wouldn't be an issue for AMP in general, since without SSR of Mustache template, you'd already know where the boundary is and can wrap the output of each rendering of the template for each item in the list in a `<div role="listitem" tabindex="0">`, but maybe I misundertood?

Reply:
NVM, re-read the change here and I think this really means that the problem exists for AMP for Email but not for website.

Reply:
You're right, this statement doesn't attribute each side effect to any specific format, even though they correspond. Hopefully by removing examples of potential consequences it will be more generally applicable.